src/deployment/android.md

```diff
@@ -336,17 +336,17 @@ to verify that the values are correct.
 : Specify the final, unique [application ID][]
```

Comment:
While you're here, consider adding punctuation to this definition entry to be consistent with the others.